### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [notriddle/budget-envelopes#51]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Budget](https://play.google.com/store/apps/details?id=com.notriddle.budget&hl=it)
3. Activate the registration from the Accessibility Scanner and use the Budget app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201119_095416](https://user-images.githubusercontent.com/25429624/99647570-44823180-2a52-11eb-8126-d1d891b94a93.png)
